reference/conanfile/attributes.rst

```diff
@@ -905,10 +905,49 @@ This object should be filled in ``package_info()`` method.
 | self.cpp_info.build_modules      | | List of relative paths to build system related utility module files created by the package. Used by   |
 |                                  | | CMake generators to export *.cmake* files with functions for consumers. Defaulted to ``[]`` (empty)   |
 +----------------------------------+---------------------------------------------------------------------------------------------------------+
+| self.cpp_info.components         | **[Experimental]** Dictionary with different components a package may have: libraries, executables...   |
++----------------------------------+---------------------------------------------------------------------------------------------------------+
 
 The paths of the directories in the directory variables indicated above are relative to the
 :ref:`self.package_folder<folders_attributes_reference>` directory.
 
+.. _cpp_info_components_attributes_reference:
+
+Components
```

Comment:
Actually I would prefer the reference the other way round:

- The attribute is only a placeholder, with a link
- The ``package_info()`` method is the section of the docs that contains all the detailed information about cpp_info and components.

Not to be changed now, but to have the idea in mind for the future.

Reply:
Ok, will take this into account for next refactors of the documentation